ql/src/test/results/clientpositive/llap/materialized_view_create_rewrite_10.q.out

```diff
@@ -79,17 +79,17 @@ STAGE PLANS:
                 TableScan
                   alias: t1
                   filterExpr: (b = 1) (type: boolean)
-                  Statistics: Num rows: 69 Data size: 13710 Basic stats: COMPLETE Column stats: NONE
+                  Statistics: Num rows: 70 Data size: #Masked# Basic stats: COMPLETE Column stats: NONE
```

Comment:
How did number of rows changed with ORC upgrade?

Reply:
That's number of stats rows that changed. Not sure how. The number of data rows is 2.